[DOCS] New default query for IM rules (#1186)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

IMPORTANT: Data in indicator indices must be <<ecs-compliant-reqs, ECS compatible>>, and so it must contain a `@timestamp` field.
+
.. *Indicator index query*: The query and filters used to filter the fields from

> the indicator index patterns. The default query (`@timestamp > "now-30d"`) checks the indicator index patterns for matched indicators that have been ingested during the past 30 days.

Reviewer: Minor quibble: `event.ingested` is the field representing ingest time; `@timestamp` captures when the event occurred (e.g. when the event originally occurred). I would maybe update the wording to reflect this definition:
```suggestion
the indicator index patterns. The default query (`@timestamp > "now-30d"`) checks the indicator index patterns for matched indicators that have occurred in the past 30 days.
```

Author: Thanks for pointing this out @rylnd! Your recommended changes are much more precise than the current wording. I'll add them into the next draft :)
